futaba/sql/models/settings.py

```diff
@@ -156,8 +208,17 @@ def __init__(self, sql, meta):
                 name="special_role_jail_not_member_check",
             ),
         )
+        self.tb_tracking_blacklists = Table(
+            "tracking_blacklists",
+            meta,
+            Column("guild_id", BigInteger, ForeignKey("guilds.guild_id"), index=True),
+            Column("type", Enum(LocationType)),
+            Column("data_id", BigInteger),
+            UniqueConstraint("guild_id", "type", "data_id"),
+        )
```

Comment:
Since we're using `LocationType` now, consider adding a `CheckConstraint()` to make sure it's only channel or user.